src/dashboard/src/templates/transfer/grid.html

```diff
@@ -191,7 +191,7 @@
           </div>
           <div style="float:left;">
             <input type='text' id='transfer-accession-number'></input>
-            <div class="help-block">Accession no.</div>
+            <div class="help-block">TMS Object ID</div>
```

Comment:
In this branch, we do need this change, so it can be tested.  We talked about refactoring the way the feature related to this line is implemented, before merging (see the 'Allow passing in a DRMC slug properly, instead of overloading the accession numebr field' checkbox in the list on the PR description